EIPS/eip-2494.md

```diff
@@ -0,0 +1,355 @@
+---
+eip: 2494
+title: Baby Jubjub Elliptic Curve
+author: Barry WhiteHat (@barryWhiteHat), Marta Bellés (@bellesmarta), Jordi Baylina (@jbaylina)
+discussions-to: https://ethereum-magicians.org/t/eip-2494-baby-jubjub-elliptic-curve/3968
+status: Draft
+type: Standards Track 
```

Comment:
```suggestion
type: Standards Track
```

This was the whitespace it complained about ^^

Reply:
Nice! It was difficult to spot, thanks!